src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
```

Comment:
The `ReturnOperation` class should be placed in the `strategy` package to follow the checklist item about not keeping all logic in a single package and to apply the Strategy pattern properly.